### PR TITLE
add env metadata edit

### PR DESCRIPTION
**Pull request by sanglevinh**

### CODE REVIEW

File: frontend/src/pages/Database/Experiments.tsx [hunk 1]
```diff
@@ -4,12 +4,17 @@ import DatabaseExperiments from "components/Database/DatabaseExperiments"
 import DatabaseWrapper from "components/Database/DatabaseWrapper"
 import { selectCurrentUser } from "store/slice/User/UserSelector"
 
+const METADATA = process.env.REACT_APP_ENABLE_EXPDB_METADATA_EDITABLE === "true"
```
Comment (itutu-tienday) [inline frontend/src/pages/Database/Experiments.tsx:7]: @sanglevinh 
Please also maintain the configuration file template (.env.example).

Comment (itutu-tienday) [inline frontend/src/pages/Database/Experiments.tsx:7]: @sanglevinh 
In my environment, even though I set `ENABLE_EXPDB_METADATA_EDITABLE=true` in .env.development, the Attribute Dialog does not seem to reflect the setting.
→ Is this a bug or my setting error?

**Review by itutu-tienday — CHANGES_REQUESTED** [review]

File: frontend/src/pages/Database/Experiments.tsx (hunk 1 above)
Comment (sanglevinh) [reply frontend/src/pages/Database/Experiments.tsx:7]: @itutu-tienday 
you should set REACT_APP_ENABLE_EXPDB_METADATA_EDITABLE = true

Comment (sanglevinh) [reply frontend/src/pages/Database/Experiments.tsx:7]: @itutu-tienday i added it, pls check it again

File: frontend/src/pages/Database/Experiments.tsx [hunk 2]
```diff
@@ -4,12 +4,17 @@ import DatabaseExperiments from "components/Database/DatabaseExperiments"
...
+
 const Experiments = () => {
   const user = useSelector(selectCurrentUser)
-
   return (
     <DatabaseWrapper>
-      <DatabaseExperiments user={user} cellPath="/console/cells" />
+      <DatabaseExperiments
```
Comment (itutu-tienday) [inline frontend/src/pages/Database/Experiments.tsx:13]: @sanglevinh 
If you are an administrative user and REACT_APP_ENABLE_EXPDB_METADATA_EDITABLE = false, not only hide the Save button, but also make the textarea readonly.

Comment (sanglevinh) [reply frontend/src/pages/Database/Experiments.tsx:13]: @itutu-tienday i fixed it, pls check it again

Comment (itutu-tienday) [reply frontend/src/pages/Database/Experiments.tsx:13]: @sanglevinh 
I have checked the modifications, OK.

File: frontend/src/components/Database/DatabaseExperiments.tsx [hunk 3]
```diff
@@ -361,15 +362,19 @@ const PopupAttributes = ({
...
+            />
             <span style={{ color: "red", display: "block" }}>{error}</span>
           </DialogContentText>
         </DialogContent>
         <DialogActions>
           <Button variant={"outlined"} autoFocus onClick={handleClose}>
             Close
           </Button>
-          {role && (
+          {role && !readonly && (
```
Comment (itutu-tienday) [inline frontend/src/components/Database/DatabaseExperiments.tsx:377]: @sanglevinh 
Please let me know when the metadata update API call is added.
- #259

**Review by itutu-tienday — CHANGES_REQUESTED** [review]

File: frontend/src/components/Database/DatabaseExperiments.tsx (hunk 3 above)
Comment (sanglevinh) [reply frontend/src/components/Database/DatabaseExperiments.tsx:377]: @itutu-tienday I have a small question, why do I use both the REACT_APP_ENABLE_EXPDB_METADATA_EDITABLE variable and the API call to check?

Comment (itutu-tienday) [reply frontend/src/components/Database/DatabaseExperiments.tsx:377]: @sanglevinh 
My instructions seemed a bit confusing, so I separated them into the following issues.

- #305

I hope the above has conveyed my intention.
If OK, I will close this thread.

Comment (sanglevinh) [reply frontend/src/components/Database/DatabaseExperiments.tsx:377]: @itutu-tienday 
What I'm understanding is that the API call of #305, and the REACT_APP_ENABLE_EXPDB_METADATA_EDITABLE variable both have the purpose of checking display metadata.
If so, I think I should only use one of the two
Or can you explain to me why we have to use both to check metadata display?

Comment (sanglevinh) [reply frontend/src/components/Database/DatabaseExperiments.tsx:377]: @itutu-tienday yes i will integrate API #305

**Review by itutu-tienday — APPROVED** [review]

**Pull request merged**